### PR TITLE
Demo session added

### PR DESCRIPTION
**Pull request by mastrogiovanni**
I added a section to dial with the need, during presentation, to show some code running. For this purpose, I added katacoda, that is useful for self-paced learning session (it start on-the-fly a virtual machine with the intended environment and can let you advance to perform a demo);
asciinema is useful to record a terminal and show during a presentation avoiding problems due to different environment.
Finally binder is useful to show notebooks in general in session related to python / machine learning / data science presentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +30,12 @@
 - [Terminalizer](https://terminalizer.com/) - Record your terminal and generate animated gif images or share a web player
 - [ttygif](https://github.com/icholy/ttygif) - Convert terminal recordings to animated gifs
 
+## Demo Tools
+*Tools that can be used to show running software during a presentation* 
+- [Katacoda](https://katacoda.com/) - Create online self-paced learning resources in virtual environment to test software. 
+- [Binder](https://mybinder.org/) - Transform a github repository into a set of interactive Jupyther Notebooks.
+- [Asciinema](https://asciinema.org/) - It helps to record terminal sessions and sharing them on the web without screen capture.
```
Comment (fsciuti) [inline README.md:32]: Move this link into Source Code Tools section (which I rename in Source Code/Terminal Tools), please!

File: README.md [hunk 2]
```diff
@@ -29,6 +30,12 @@
 - [Terminalizer](https://terminalizer.com/) - Record your terminal and generate animated gif images or share a web player
 - [ttygif](https://github.com/icholy/ttygif) - Convert terminal recordings to animated gifs
 
+## Demo Tools
+*Tools that can be used to show running software during a presentation* 
+- [Katacoda](https://katacoda.com/) - Create online self-paced learning resources in virtual environment to test software. 
+- [Binder](https://mybinder.org/) - Transform a github repository into a set of interactive Jupyther Notebooks.
```
Comment (fsciuti) [inline README.md]: Move this link into Other Tools section, please!

File: README.md [hunk 3]
```diff
@@ -29,6 +30,12 @@
 - [Terminalizer](https://terminalizer.com/) - Record your terminal and generate animated gif images or share a web player
 - [ttygif](https://github.com/icholy/ttygif) - Convert terminal recordings to animated gifs
 
+## Demo Tools
+*Tools that can be used to show running software during a presentation* 
+- [Katacoda](https://katacoda.com/) - Create online self-paced learning resources in virtual environment to test software.
```
Comment (fsciuti) [inline README.md]: Can you create a new Learning Platform section, please?
You can put this link into this new section!

File: README.md (hunk 1 above)
Comment (mastrogiovanni) [reply README.md:32]: ok

File: README.md (hunk 2 above)
Comment (mastrogiovanni) [reply README.md]: ok

File: README.md (hunk 3 above)
Comment (mastrogiovanni) [reply README.md]: ok

**Review by mastrogiovanni — COMMENTED** [review]
I moved:
- katacoda in new "Learning Platforms"
- binder in ""Other tools"
- Asciinema in "Source Code Tools"

**Comment by fsciuti** [comment]
Thank you so much @mastrogiovanni!!!!! :)

**Pull request merged**